File: happykopiAPI/happykopiAPI/DTOs/Modifier/Outgoing Data/ModifierDetailsDto.cs
namespace happykopiAPI.DTOs.Modifier.Outgoing_Data
{
    public class ModifierDetailsDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Price { get; set; }
        public string Type { get; set; }
        public bool IsAvailable { get; set; }
        public List<ModifierStockItemDetailsDto> LinkedItems { get; set; }
    }
}

File: happykopiApp/src/app/core/dtos/modifier/modifier-details-dto.ts
import { ModifierLinkStockItem } from "./modifier-link-stock-item.model";

export interface ModifierDetailsDto {
    id: number;
    name: string;
    price: number;
    type: number;
    isAvailable: boolean;
    isActive: boolean;
    ozAmount?: number;
    linkedItems?: ModifierLinkStockItem[];
}
